bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -794,9 +799,22 @@ Full pathname to the inventory initialization control file.
 <entry id="use_fates_luh" type="logical" category="physics"
        group="clm_inparm" valid_values="" >
 If TRUE, enable use of land use harmonization (LUH) state and transition data from luh_timeseries file.
+This is enabled by default if fates_harvest_mode is set to use the raw LUH2 harvest data
 (Also, only valid for use_fates = true and is incompatible with transient runs currently.)
```

Comment:
Is this line still true about being incompatible with transient runs? By transient does this means a HIST compset? We obviously are going to want HIST*FATES compsets to work.